src/index.html

```diff
@@ -15,8 +15,111 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossoriginn="anonymus"
```

Comment:
The `crossoriginn` attribute is misspelled. It should be `crossorigin` to ensure proper handling of CORS requests.